test/test_images/probe_helper/main.go

```diff
@@ -88,6 +100,9 @@ type envConfig struct {
 	// Environment variable containing the CloudStorageSource Bucket ID that objects will be written to by the probeHelper for the CloudStorageSource probe
 	CloudStorageSourceBucketID string `envconfig:"CLOUDSTORAGESOURCE_BUCKET_ID" default:"cloudstoragesource-bucket"`
 
+	// Environment variable containing the duration between events emitted by the CloudSchedulerSource
+	CloudSchedulerSourcePeriod time.Duration `envconfig:"CLOUDSCHEDULERSOURCE_PERIOD" default:"1m"`
```

Comment:
nit: if the scheduler source fires per minute, we should give it a bit buffer here. The default may be 90s. or even 2m?

Reply:
Done. Picked 90s.